docs/getting-started-guides/docker-multinode.md

```diff
@@ -0,0 +1,114 @@
+Running Multi-Node Kubernetes Using Docker
```

Comment:
The file must open with these lines:

---
---

Do not put the title -- that will be pulled in from guides.yml